### PR TITLE
EAPI5 cleanup

### PR DESCRIPTION
**Pull request by jsmolic**

### CODE REVIEW

**Review by a17r — APPROVED** [review]
ack re @gentoo/office packages.

File: dev-db/libodbc++/libodbc++-0.2.5-r3.ebuild [hunk 1]
```diff
@@ -91,17 +90,17 @@ src_compile() {
 }
 
 src_install() {
-	dodoc AUTHORS BUGS ChangeLog NEWS README THANKS TODO
+	einstalldocs
 
 	buildlist="${SB} ${SB_MT}"
 	#use qt3 && buildlist="${buildlist} $SB_QT $SB_QT_MT"
 	for sd in ${buildlist}; do
```
Comment (thesamesam) [inline dev-db/libodbc++/libodbc++-0.2.5-r3.ebuild:97]: Nit: local sd, ${} for references to it

File: dev-db/libodbc++/libodbc++-0.2.5-r3.ebuild [hunk 2]
```diff
@@ -91,17 +90,17 @@ src_compile() {
...
 		einfo "Doing install pass for $sd"
-		cd ${sd}
+		cd ${sd} || die
 		emake DESTDIR="${D}" install
 	done
 	if [[ "${P}" != "${PF}" ]]; then
-		mv "${D}"/usr/share/doc/${P}/* "${D}"/usr/share/doc/${PF}/
-		rmdir "${D}"/usr/share/doc/${P}
+		mv "${D}"/usr/share/doc/${P}/* "${D}"/usr/share/doc/${PF}/ || die
```
Comment (thesamesam) [inline dev-db/libodbc++/libodbc++-0.2.5-r3.ebuild:103]: ED

File: dev-libs/bareos-fastlzlib/bareos-fastlzlib-0_pre20150524.ebuild [hunk 3]
```diff
@@ -1,30 +1,28 @@
...
-RDEPEND="${DEPEND}
+RDEPEND="
 	!app-arch/lz4
 "
-
-S="${WORKDIR}/fastlzlib-master"
+BDEPEND="app-arch/unzip"
 
 src_configure() {
 	econf --disable-static
```
Comment (thesamesam) [inline dev-libs/bareos-fastlzlib/bareos-fastlzlib-0_pre20150524.ebuild:21]: Not needed anymore with 8, yay!

File: mail-filter/popfile/popfile-1.1.3-r1.ebuild [hunk 4]
```diff
@@ -29,21 +31,19 @@ RDEPEND="virtual/perl-Digest-MD5
...
-DEPEND="app-arch/unzip"
-
-S="${WORKDIR}"
+BDEPEND="app-arch/unzip"
 
 src_prepare() {
+	default
 	local f
-	for f in `find ./ -type f`; do
+	for f in $(find ./ -type f || die); do
```
Comment (thesamesam) [inline mail-filter/popfile/popfile-1.1.3-r1.ebuild:39]: We should really grep for backticks and find other cases of these

File: net-misc/telnet-bsd/telnet-bsd-1.2-r2.ebuild [hunk 5]
```diff
@@ -38,12 +41,10 @@ src_configure() {
 }
```
Comment (thesamesam) [inline net-misc/telnet-bsd/telnet-bsd-1.2-r2.ebuild:42]: Would be nice if append-libs worked instead but not tried.

File: net-misc/telnet-bsd/telnet-bsd-1.2-r2.ebuild [hunk 6]
```diff
@@ -17,13 +17,16 @@ IUSE="nls xinetd"
 RDEPEND="sys-libs/ncurses:="
 DEPEND="${RDEPEND}
 	!net-misc/netkit-telnetd
-	xinetd? ( sys-apps/xinetd )
-	virtual/pkgconfig"
+	xinetd? ( sys-apps/xinetd )"
```
Comment (thesamesam) [inline net-misc/telnet-bsd/telnet-bsd-1.2-r2.ebuild:20]: Also, surely this is needed for at least runtime?

File: net-misc/telnet-bsd/telnet-bsd-1.2-r2.ebuild (hunk 5 above)
Comment (jsmolic) [reply net-misc/telnet-bsd/telnet-bsd-1.2-r2.ebuild:42]: Yeah looks like append-libs was already there at some point, but removed due to bug https://bugs.gentoo.org/559154

File: dev-db/libodbc++/libodbc++-0.2.5-r3.ebuild (hunk 1 above)
Comment (jsmolic) [reply dev-db/libodbc++/libodbc++-0.2.5-r3.ebuild:98]: done

File: mail-filter/popfile/popfile-1.1.3-r1.ebuild (hunk 4 above)
Comment (jsmolic) [reply mail-filter/popfile/popfile-1.1.3-r1.ebuild:39]: Yeah, I did check and there are couple more cases

File: dev-libs/bareos-fastlzlib/bareos-fastlzlib-0_pre20150524.ebuild (hunk 3 above)
Comment (jsmolic) [reply dev-libs/bareos-fastlzlib/bareos-fastlzlib-0_pre20150524.ebuild]: yup, forgot to remove it :D

File: net-misc/telnet-bsd/telnet-bsd-1.2-r2.ebuild (hunk 6 above)
Comment (jsmolic) [reply net-misc/telnet-bsd/telnet-bsd-1.2-r2.ebuild]: :thinking: you're right, I think this should be RDEPEND only

File: dev-db/libodbc++/libodbc++-0.2.5-r3.ebuild [hunk 7]
```diff
@@ -24,27 +23,26 @@ SB_MT="${S}-build-mt"
...
 
 	# Fix undeclared ODBCXX_STRING_PERCENT symbol, bug #532356
 	sed -i -e 's/ODBCXX_STRING_PERCENT/"%"/' src/dtconv.h || die
-
-	epatch_user
 }
 
 src_configure() {
 	local commonconf buildlist
 	commonconf="--with-odbc=/usr --without-tests"
```
Comment (SoapGentoo) [inline dev-db/libodbc++/libodbc++-0.2.5-r3.ebuild:44]:
```suggestion
	commonconf="--with-odbc=${EPREFIX}/usr --without-tests"
```

File: dev-db/libodbc++/libodbc++-0.2.5-r3.ebuild [hunk 8]
```diff
@@ -53,10 +51,11 @@ src_configure() {
...
+	local sd
 	for sd in ${buildlist}; do
 		einfo "Doing configure pass for $sd"
-		mkdir -p "${sd}"
-		cd "${sd}"
+		mkdir -p "${sd}" || die
+		cd "${sd}" || die
 		commonconf2=''
 		LIBS=''
 		[ "${sd}" == "${SB_MT}" -o "${sd}" == "${SB_QT_MT}" ] && commonconf2="${commonconf2} --enable-threads"
```
Comment (SoapGentoo) [inline dev-db/libodbc++/libodbc++-0.2.5-r3.ebuild:61]: bash brackets?

File: dev-db/libodbc++/libodbc++-0.2.5-r3.ebuild [hunk 9]
```diff
@@ -79,29 +78,30 @@ src_configure() {
...
 src_compile() {
-	local buildlist failures
+	local buildlist failures sd
 	buildlist="${SB} ${SB_MT}"
 	#use qt3 && buildlist="${buildlist} $SB_QT $SB_QT_MT"
 	for sd in ${buildlist}; do
-		einfo "Doing compile pass for $sd"
+		einfo "Doing compile pass for ${sd}"
 		cd "${sd}"
 		emake LIBS='' || failures="${failures} ${sd//${S}-}"
```
Comment (SoapGentoo) [inline dev-db/libodbc++/libodbc++-0.2.5-r3.ebuild:87]: `emake -C "${sd}"` ?

File: dev-db/libodbc++/libodbc++-0.2.5-r3.ebuild [hunk 10]
```diff
@@ -79,29 +78,30 @@ src_configure() {
...
 	done
 	[ -n "${failures}" ] && die "Failures: ${failures}"
```
Comment (SoapGentoo) [inline dev-db/libodbc++/libodbc++-0.2.5-r3.ebuild:89]: bash
```suggestion
	[[ -n ${failures} ]] && die "Failures: ${failures}"
```

File: net-misc/telnet-bsd/telnet-bsd-1.2-r2.ebuild (hunk 5 above)
Comment (thesamesam) [reply net-misc/telnet-bsd/telnet-bsd-1.2-r2.ebuild:43]: eh, okay, let's not worry about it. add the bug reference above it though. Also, why are we calling emake in src_configure!?

File: net-im/telepathy-connection-managers/telepathy-connection-managers-2-r2.ebuild [hunk 11]
```diff
@@ -1,19 +1,17 @@
...
 DESCRIPTION="Meta-package for Telepathy Connection Managers"
 HOMEPAGE="https://telepathy.freedesktop.org/"
-SRC_URI=""
+
 LICENSE="metapackage"
 SLOT="0"
-
 KEYWORDS="~alpha amd64 ~arm arm64 ~ia64 ~ppc ~ppc64 ~sparc x86 ~x86-linux"
 
 IUSE="gadu icq +irc meanwhile msn sip sipe +xmpp yahoo steam zeroconf"
```
Comment (thesamesam) [inline net-im/telepathy-connection-managers/telepathy-connection-managers-2-r2.ebuild:13]: sort?

File: net-misc/telnet-bsd/telnet-bsd-1.2-r2.ebuild (hunk 5 above)
Comment (jsmolic) [reply net-misc/telnet-bsd/telnet-bsd-1.2-r2.ebuild:43]: uhhh I removed that atrocity but somehow got lost while amending/rebasing

File: dev-db/libodbc++/libodbc++-0.2.5-r3.ebuild (hunk 7 above)
Comment (jsmolic) [reply dev-db/libodbc++/libodbc++-0.2.5-r3.ebuild]: done

File: dev-db/libodbc++/libodbc++-0.2.5-r3.ebuild (hunk 8 above)
Comment (jsmolic) [reply dev-db/libodbc++/libodbc++-0.2.5-r3.ebuild]: done

File: dev-db/libodbc++/libodbc++-0.2.5-r3.ebuild (hunk 9 above)
Comment (jsmolic) [reply dev-db/libodbc++/libodbc++-0.2.5-r3.ebuild]: :+1:  :wink:

**Pull request closed without merging**